lib/moip2/accounts_api.rb

```diff
@@ -13,5 +13,15 @@ def base_path
     def create(account)
       Resource::Account.new client, client.post(base_path, account)
     end
+
+    def exists?(tax_document = "")
+      response = client.get("#{base_path}/exists?tax_document=#{tax_document}")
```

Comment:
Quando for fazer o release daí adicionaremos na documentação essa informação, pra nao causar confusão